### DISCUSSION TITLE
What does this Error Code 1 mean? /@npmcli/promise-spawn/index.js:64:27

### DISCUSSION THREAD

**Discussion by AlphaJuliettOmega** [npm/feedback#209] [CLI] [answered]
It looks like Node fails to spawn worker threads, but that's just a guess.

```
22 verbose stack Error: command failed
22 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/@npmcli/promise-spawn/index.js:64:27)
22 verbose stack     at ChildProcess.emit (node:events:378:20)
22 verbose stack     at maybeClose (node:internal/child_process:1067:16)
22 verbose stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)
...
25 verbose Darwin 20.1.0
26 verbose argv "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/bin/npm-cli.js" "exec" "--" "react-native" "run-ios"
27 verbose node v15.8.0
28 verbose npm  v7.5.3
29 error code 1
```
on run of `npx react-native run-ios`

Is there anything I can do to get a better error message?
Error code 1 does not help me.

Is it related to this closed but unresolved thread?
https://github.com/npm/npm/issues/3139

**Comment by MylesBorins** [comment, marked as answer]
@AlphaJuliettOmega thanks for stopping by.

This feedback discussions repo is for discussing new features for npm, not for technical support. It seems like your best bet would be to open an issue over on the react-native repo to get support.
